dominion/frontend/src/assets/App.css

```diff
@@ -0,0 +1,94 @@
+.top-menu {
```

Comment:
Styles should be with their respective components... *If* (and that's a very big if) we need generic `css` file, please explain why and note it in code.

Reply:
@kilemensi Added a comment on why I used css for this